apps/journeys-admin/src/components/Editor/Slider/Content/Goals/Goals.spec.tsx

```diff
@@ -167,19 +166,19 @@ describe('GoalsList', () => {
           variant: 'admin'
         }}
       >
-        <GoalsList goals={[]} />
+        <Goals />
       </JourneyProvider>
     )
     expect(getByText('The Journey Goals')).toBeInTheDocument()
 
     expect(getAllByText('https://m.me/some_user')[0]).toBeInTheDocument()
```

Comment:
```suggestion
    expect(getByText('https://m.me/some_user')).toBeInTheDocument()
```
Can these also be changed to getByText?